src/combat.ts

```diff
@@ -11,17 +12,30 @@ export default class Macro extends StrictMacro {
         `!mpbelow ${mpCost($skill`Stuffed Mortar Shell`)}`,
         Macro.trySkill($skill`Stuffed Mortar Shell`)
       );
-    return (useCinch ? macroHead.trySkill($skill`Cincho: Confetti Extravaganza`) : macroHead)
-      .while_(
-        `!mpbelow ${mpCost($skill`Saucegeyser`)} && hasskill ${toInt($skill`Saucegeyser`)}`,
-        Macro.skill($skill`Saucegeyser`)
-      )
-      .while_(
-        `!mpbelow ${mpCost($skill`Saucestorm`)} && hasskill ${toInt($skill`Saucestorm`)}`,
-        Macro.skill($skill`Saucestorm`)
-      )
-      .attack()
-      .repeat();
+    if (myPrimestat() === $stat`Mysticality`) {
+      //Update to add alternative combat options for non-Mys classes
+      return (useCinch ? macroHead.trySkill($skill`Cincho: Confetti Extravaganza`) : macroHead)
+        .while_(
+          `!mpbelow ${mpCost($skill`Saucegeyser`)} && hasskill ${toInt($skill`Saucegeyser`)}`,
+          Macro.skill($skill`Saucegeyser`)
+        )
+        .while_(
+          `!mpbelow ${mpCost($skill`Saucestorm`)} && hasskill ${toInt($skill`Saucestorm`)}`,
+          Macro.skill($skill`Saucestorm`)
+        )
```

Comment:
This should be the default for all non-mus classes, we probably shouldn't be using Lunging Thrust-Smack for mox classes.

Also, this should be refactored into something like
```
const defaultClassAttack = mainStat === $stat`Muscle` ? Macro.while_(/*LTS logic*/) : Macro.while(/*spellslinging logic/*);
return (useCinch ? macroHead.trySkill($skill`Cincho: Confetti Extravaganza`) : macroHead).defaultClassAttack.attack().repeat()
```